registry/storage/garbagecollect.go

```diff
@@ -47,6 +49,17 @@ func MarkAndSweep(ctx context.Context, storageDriver driver.StorageDriver, regis
 		}
 
 		err = manifestEnumerator.Enumerate(ctx, func(dgst digest.Digest) error {
+			if manifestswithouttags {
+				tags, errr := repository.Tags(ctx).Lookup(ctx, distribution.Descriptor{Digest: dgst})
+				if errr != nil {
+					return fmt.Errorf("failed to retrieve tags for digest %v: %v", dgst, errr)
+				}
+				if len(tags) == 0 {
+					emit("manifest has no tags %s@%s", repoName, dgst)
+					manifestSet[dgst] = repoName
```

Comment:
I did not fully understand this. How I should check it with paths.go? And you mean that I could delete manifest here already? (means that I need initialize vacuum in start of function)